modules/tests/test/copy_template_test.go

```diff
@@ -94,66 +81,39 @@ var _ = Describe("Copy template task", func() {
 				TaskData: testconfigs.CopyTemplateTaskData{
 					SourceTemplateName: testtemplate.CirrosTemplateName,
 					TargetTemplateName: NewTemplateName,
 					Template:           testtemplate.NewCirrosServerTinyTemplate().Build(),
 				},
 			}),
-			Entry("should create template in different namespace", &testconfigs.CopyTemplateTestConfig{
-				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
-					ServiceAccount: CopyTemplateServiceAccountName,
-				},
-				TaskData: testconfigs.CopyTemplateTaskData{
-					SourceTemplateName:      testtemplate.CirrosTemplateName,
-					TargetTemplateName:      NewTemplateName,
-					TargetTemplateNamespace: DeployTargetNS,
-					Template:                testtemplate.NewCirrosServerTinyTemplate().Build(),
-				},
-			}),
 			Entry("no target template name specified", &testconfigs.CopyTemplateTestConfig{
-				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
-					ServiceAccount: CopyTemplateServiceAccountName,
-				},
+				TaskRunTestConfig: testconfigs.TaskRunTestConfig{},
 				TaskData: testconfigs.CopyTemplateTaskData{
 					SourceTemplateName: testtemplate.CirrosTemplateName,
 					Template:           testtemplate.NewCirrosServerTinyTemplate().Build(),
 				},
 			}),
 			Entry("no target namespaces specified", &testconfigs.CopyTemplateTestConfig{
-				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
-					ServiceAccount: CopyTemplateServiceAccountName,
-				},
+				TaskRunTestConfig: testconfigs.TaskRunTestConfig{},
 				TaskData: testconfigs.CopyTemplateTaskData{
-					SourceTemplateName:      testtemplate.CirrosTemplateName,
-					TargetTemplateNamespace: EmptyTargetNS,
-					Template:                testtemplate.NewCirrosServerTinyTemplate().Build(),
+					SourceTemplateName: testtemplate.CirrosTemplateName,
+					Template:           testtemplate.NewCirrosServerTinyTemplate().Build(),
 				},
 			}),
 			Entry("no source namespaces specified", &testconfigs.CopyTemplateTestConfig{
-				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
-					ServiceAccount: CopyTemplateServiceAccountName,
-				},
+				TaskRunTestConfig: testconfigs.TaskRunTestConfig{},
 				TaskData: testconfigs.CopyTemplateTaskData{
-					SourceTemplateName:      testtemplate.CirrosTemplateName,
-					SourceTemplateNamespace: EmptyTargetNS,
-					TemplateNamespace:       DeployTargetNS,
-					Template:                testtemplate.NewCirrosServerTinyTemplate().Build(),
+					SourceTemplateName: testtemplate.CirrosTemplateName,
+					Template:           testtemplate.NewCirrosServerTinyTemplate().Build(),
 				},
 			}),
 			Entry("no namespaces specified", &testconfigs.CopyTemplateTestConfig{
-				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
-					ServiceAccount: CopyTemplateServiceAccountName,
-				},
+				TaskRunTestConfig: testconfigs.TaskRunTestConfig{},
 				TaskData: testconfigs.CopyTemplateTaskData{
-					SourceTemplateName:      testtemplate.CirrosTemplateName,
-					SourceTemplateNamespace: EmptyTargetNS,
```

Comment:
Why are all namespaces not required anymore? No permissions?

Reply:
yes